api/app/signals/apps/email_integrations/tests/test_actions.py

```diff
@@ -172,6 +172,42 @@ def test_send_email_fails(self, mocked):
         self.assertFalse(self.action(signal, dry_run=False))
         self.assertEqual(len(mail.outbox), 0)
 
+    def test_allowed_contact_false(self):
```

Comment:
Change the test so that it will only trigger when the feedback has been submitted. And add a tests to see if it will not be triggered when there is a feedback not submitted.